services/core/shared/core/compat/sdk_v5/accountIndex.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const Redis = require('ioredis');
+
+const Queue = require('../../queue');
+
+const {
+	getTableInstance,
+} = require('../../../indexdb/mysql');
+
+const {
+	getAccountsByAddress,
+	getAccountsByPublicKey2,
+} = require('./accounts');
+
+const config = require('../../../../config');
+
+const redis = new Redis(config.endpoints.redis);
+
+const accountsIndexSchema = require('./schema/accounts');
+
+const getAccountIndex = () => getTableInstance('accounts', accountsIndexSchema);
+
+const updateAccountInfoPk = async (job) => {
+	const publicKey = job.data;
+
+	const account = await getAccountsByPublicKey2([publicKey]);
+	if (account.length) {
+		const accountsDB = await getAccountIndex();
+		await accountsDB.upsert(account);
+	}
```

Comment:
Please consider updating the variable name to `accounts` since `getAccountsByPublicKey2` returns an array of accounts.
```suggestion
	const accounts = await getAccountsByPublicKey2([publicKey]);
	if (accounts.length) {
		const accountsDB = await getAccountIndex();
		await accountsDB.upsert(accounts);
	}
```